src/StaticWebAssetsSdk/Tasks/DefineStaticWebAssets.cs

```diff
@@ -110,6 +110,17 @@ public override bool Execute()
                 if (SourceType == StaticWebAsset.SourceTypes.Discovered)
                 {
                     var candidateMatchPath = GetDiscoveryCandidateMatchPath(candidate);
+                    if (Path.IsPathRooted(candidateMatchPath) && candidateMatchPath == candidate.ItemSpec)
+                    {
+                        var normalizedAssetPath = Path.GetFullPath(candidate.GetMetadata("FullPath"));
+                        var normalizedDirectoryPath = Path.GetDirectoryName(BuildEngine.ProjectFileOfTaskNode);
+                        if (normalizedAssetPath.StartsWith(normalizedDirectoryPath))
+                        {
+                            var result = normalizedAssetPath.Substring(normalizedDirectoryPath.Length);
```

Comment:
Wouldn't the `normalizedAssetPath` start with `\` (in some cases)?

The `Path.GetDirectoryName` will return path without trailing slash, so subtracting it from file path won't remove the the last slash before file name, will it?

Reply:
@maraf fair enough.

Better safe than sorry. It's unfortunate we can't use Path.GetRelativePath :(